### PR TITLE
Drop support for python3.6

### PR DESCRIPTION
**Pull request by Gobot1234**

### CODE REVIEW

**Comment by MichaelTiemannOSC** [comment]
By merging this, it would unblock #451, which would in turn make that issue labeled "good first issue" an actual good first issue.  It's not a great first issue if it's blocked by paradoxical CI/CD rules.

**Comment by Gobot1234** [comment]
As much as I would love to merge it, I don't have permissions to do so

**Review by cetanu — APPROVED** [review]
Seems fine, people stuck on 3.6 can use the previous version and it's advisable that they upgrade anyway.

**Pull request merged**
